CHANGELOG.md

```diff
@@ -1,2688 +1,2830 @@
-# Changelog
```

Comment:
You'll need to review this file with "Hide whitespace" enabled. I think something happened with the line endings, maybe Windows <--> Unix line endings?

Reply:
I think the issues is that it's adding a _load_ more newlines! Specifically straight after every github username 🤔

Reply:
Question of why is a good one: @greg-finley ?

Reply:
At some point, I recall seeing `^M` in the file. That's a "carriage return", which on Windows are often accompanied by newlines. Not sure why they'd suddenly appear now, unless there's screen scraping involved in creating the file, perhaps?